From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Chris Bobbe <cbobbe@zulip.com>
Date: Wed, 20 Jul 2022 14:38:41 -0700
Subject: [shortcut] expo: Fudge expo-modules-core export in useless-seeming
 modules
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

I'm not sure what these files are for; the types they define aren't
made available in an
  import … from 'expo-application'
, etc., and I don't see a reason we'd ever do an
  import … from 'expo-application/build/ExpoApplication
.

As mentioned in the code comments, if it seems useful, we could
eventually get types for expo-modules-core, probably with TsFlower.
---
 types/expo-application/build/ExpoApplication.js.flow          | 4 +++-
 .../build/ExpoScreenOrientation.js.flow                       | 4 +++-
 2 files changed, 6 insertions(+), 2 deletions(-)

diff --git types/expo-application/build/ExpoApplication.js.flow types/expo-application/build/ExpoApplication.js.flow
index 99d6241d3..b1e5a32af 100644
--- types/expo-application/build/ExpoApplication.js.flow
+++ types/expo-application/build/ExpoApplication.js.flow
@@ -1,6 +1,8 @@
 /* @flow
  * @generated by TsFlower
  */
+/* $FlowFixMe[untyped-type-import] - Could get types for expo-modules-core,
+   if enough of types/expo-* would make good use of them. */
 import type { ProxyNativeModule as $tsflower_import_type$_$expo_2d_modules_2d_core$ProxyNativeModule } from 'expo-modules-core';
-declare var _default: $tsflower_import_type$_$expo_2d_modules_2d_core$ProxyNativeModule;
+declare var _default: $FlowFixMe; // $tsflower_import_type$_$expo_2d_modules_2d_core$ProxyNativeModule
 export default _default;
diff --git types/expo-screen-orientation/build/ExpoScreenOrientation.js.flow types/expo-screen-orientation/build/ExpoScreenOrientation.js.flow
index 99d6241d3..b1e5a32af 100644
--- types/expo-screen-orientation/build/ExpoScreenOrientation.js.flow
+++ types/expo-screen-orientation/build/ExpoScreenOrientation.js.flow
@@ -1,6 +1,8 @@
 /* @flow
  * @generated by TsFlower
  */
+/* $FlowFixMe[untyped-type-import] - Could get types for expo-modules-core,
+   if enough of types/expo-* would make good use of them. */
 import type { ProxyNativeModule as $tsflower_import_type$_$expo_2d_modules_2d_core$ProxyNativeModule } from 'expo-modules-core';
-declare var _default: $tsflower_import_type$_$expo_2d_modules_2d_core$ProxyNativeModule;
+declare var _default: $FlowFixMe; // $tsflower_import_type$_$expo_2d_modules_2d_core$ProxyNativeModule
 export default _default;
-- 
2.32.0

